modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/Commenting/default.js

```diff
@@ -135,13 +135,23 @@
                 });
             }
 
+            function focusOnHash($container) {
+                if (location.hash === ''){
+                    return;
+                }
+
+                $('html, body').animate({ scrollTop: $(location.hash).offset().top + $(this).height() / 2 }, 'slow');
```

Comment:
I think this is very generic code. We should move this to a framework package, or restrict it: apply it only for hashes start with `cms-comment`.

Also, doesn it cover the "reply" scenario (should focus on the reply)

Reply:
> Also, doesn it cover the "reply" scenario (should focus on the reply)

It can't focus on reply or a single comment because ReturnUrlHash only has EntityType & EntityId information. So it focuses on the whole related comment area.
Of course If focusing on a reply or single comment is very important, i can do it.